assets/js/script.js

```diff
@@ -0,0 +1,107 @@
+import { UI } from "./ui.js";
+const http = new Http();
```

Comment:
You don't seem to store anything in a class instance, I'd say just export either an object with all the possible fetch method functions or even all the functions individually so that you could use any of those anywhere you want. 😃